### PR TITLE
qBittorrent.qBittorrent version 4.3.6

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19135)

### CODE REVIEW

**Comment by KaranKad** [comment]
image
It is a fairly popular application but smartscreen will warn you until many people download it.

**Comment by OfficialEsco** [comment]
@denelon it would be nice if someone from the MS team would answer/check these PRs instead of having to be told to do so..

**Comment by denelon** [comment]
@OfficialEsco We're working to improve the validation processes.  When the error is something we need to look at, it should get assigned to one of the engineers.

This one looks like it's triggering SmartScreen from the image @KaranKad provided. It is possible to submit sites to SmartScreen and installers to Defender for analysis. I'll see if I can get confirmation on this one.

**Comment by palenshus** [comment]
Looks like there's a hash mismatch again, but in addition it looks like the Defender scan is failing for them as well

File: manifests/q/qBittorrent/qBittorrent/4.3.6/qBittorrent.qBittorrent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  Scope: machine
+  InstallerUrl: https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.3.6/qbittorrent_4.3.6_x64_setup.exe/download
+  InstallerSha256: 9b2474f119a21e8b65f600f3069a5eb998974dd5d05df6003af3ff7b26abc5b5
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.3.6/qbittorrent_4.3.6_setup.exe/download
+  InstallerSha256: c2f9eb2f702bd79e1318f614c5e43c1234862eea26c9e4ae0292f1e7e0fd9cbc
```
Comment (OfficialEsco) [inline manifests/q/qBittorrent/qBittorrent/4.3.6/qBittorrent.qBittorrent.installer.yaml:28]:
```suggestion
  InstallerSha256: 9B2474F119A21E8B65F600F3069A5EB998974DD5D05DF6003AF3FF7B26ABC5B5
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.3.6/qbittorrent_4.3.6_setup.exe/download
  InstallerSha256: C2F9EB2F702BD79E1318F614C5E43C1234862EEA26C9E4AE0292F1E7E0FD9CBC
```

**Review by OfficialEsco — COMMENTED** [review]
The hash matches.

Also VirtusTotal results:
x64 link: https://www.virustotal.com/gui/url/6d3c5a2382cf3c3f22bd3daf86721464937ed0ed068e854a44b5e32a9b791349/detection
x64 file: https://www.virustotal.com/gui/file/9b2474f119a21e8b65f600f3069a5eb998974dd5d05df6003af3ff7b26abc5b5/detection
x86 link: https://www.virustotal.com/gui/url/dcf05e9623f7229b5b3f8f72174ce17e5b2e4144dfb2ecfeed6f868b5b43a910/detection
x86 file: https://www.virustotal.com/gui/file/c2f9eb2f702bd79e1318f614c5e43c1234862eea26c9e4ae0292f1e7e0fd9cbc/detection

None of them got SmartScreen warning in Edge or when executing

**Comment by OfficialEsco** [comment]
Cute message, however wrong detection

```
7/15/2021 1:26:13 PM
Status: Completed
##[error] Installer https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.3.6/qbittorrent_4.3.6_x64_setup.exe/download Scan information: 'Error information: BlockingDetectionFound'
##[error] Result: Failed

7/15/2021 1:26:18 PM
Status: Completed
##[error] Installer https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.3.6/qbittorrent_4.3.6_setup.exe/download Scan information: 'Error information: BlockingDetectionFound'
##[error] Result: Failed
```

It's probably marked as PUP in Windows Defender, but it's somehow allowed to execute in my environment.

**Comment by denelon** [comment]
What results does the [Microsoft Security Intelligence](https://www.microsoft.com/en-us/wdsi/filesubmission) report provide?

**Comment by KaranKad** [comment]
x64 version is the one detected as PUA
image

**Comment by denelon** [comment]
@KaranKad at least we know why it's failing now. I'm not sure how long it takes after submission to get the review so it isn't marked as PUA (assuming it's not).

**Comment by KaranKad** [comment]
It's only detected as PUA when downloading from Edge, it installs without any smartscreen popup in sandbox unlike before.
image

**Comment by OfficialEsco** [comment]
Something weird here, cuz i am not getting any SmartScreen alert at work, home, sandbox or vm...
https://www.microsoft.com/en-us/wdsi/submission/77197cb8-fd0f-4135-9284-8cc5d8daa6e6
This is the one i submitted yesterday but all it says is
`The details for the submission were not found or the submission has expired.`
and
`Status: In progress` in Histroy

**Comment by denelon** [comment]
@OfficialEsco I don't have permissions to view the contents at that URL. I'll see if I can get one of the engineers to trace this one down if it doesn't pass with this run. I'm also a bit confused at the moment. This is somewhere in one of the defense in depth layers that likely isn't exposing sufficient meta-data in the responses bubbled up to easily trace.

**Comment by denelon** [comment]
Looking into this one.

**Comment by OfficialEsco** [comment]
Do you guys like just not like qBittorrent? 🤔

image
image

**Comment by denelon** [comment]
I think one of the installers (I can't remember if it's x86 or x64) has a PUA on it with one of the analysis services.

**Comment by OfficialEsco** [comment]
Its the x64 https://github.com/microsoft/winget-pkgs/pull/19135#issuecomment-880929563
The one i've downloaded multiple times without any warnings and sent in for review a week ago..

**Comment by OfficialEsco** [comment]
Still
image
but for once i got this
image


@KaranKad did you forget to reopen the PR btw?

**Comment by OfficialEsco** [comment]
Ahh i see, hopefully there is a new version soon and that it passed Microsoft Defender/SmartScreen

**Comment by KaranKad** [comment]
The thing is Defender does not detect it as PUA now and does not block installation when I tested it using Sandbox Test. Although Edge blocks it. VirusTotal Reports 0/68, so I doubt that Sandbox machines used in validation process run old defender definitions. idk 🤷‍♂️

**Pull request closed without merging**